includes/Admin/Settings_Screens/Product_Sync.php

```diff
@@ -59,19 +66,22 @@ public function enqueue_assets() {
 		wp_enqueue_script( 'facebook-for-woocommerce-modal', plugins_url( '/facebook-for-woocommerce/assets/js/facebook-for-woocommerce-modal.min.js' ), [ 'jquery', 'wc-backbone-modal', 'jquery-blockui' ], \WC_Facebookcommerce::PLUGIN_VERSION );
 		wp_enqueue_script( 'facebook-for-woocommerce-settings-sync', plugins_url( '/facebook-for-woocommerce/assets/js/admin/facebook-for-woocommerce-settings-sync.min.js' ), [ 'jquery', 'wc-backbone-modal', 'jquery-blockui', 'facebook-for-woocommerce-modal', 'wc-enhanced-select' ], \WC_Facebookcommerce::PLUGIN_VERSION );
 
-		/* translators: Placeholders %1$s - opening <strong> html tag, %2$s closing </strong> html tag, {count} number of remaining items */
-		$sync_remaining_items_string = _n_noop( '%1$sProgress:%2$s {count} item remaining.', '%1$sProgress:%2$s {count} items remaining.', 'facebook-for-woocommerce' );
+		/* translators: Placeholders: {count} number of remaining items */
+		$sync_remaining_items_string = _n_noop( '{count} item remaining.', '{count} items remaining.', 'facebook-for-woocommerce' );
 
 		wp_localize_script( 'facebook-for-woocommerce-settings-sync', 'facebook_for_woocommerce_settings_sync', [
 			'ajax_url'                        => admin_url( 'admin-ajax.php' ),
 			'set_excluded_terms_prompt_nonce' => wp_create_nonce( 'set-excluded-terms-prompt' ),
+			'sync_products_nonce'             => wp_create_nonce( self::ACTION_SYNC_PRODUCTS ),
+			'sync_status_nonce'               => wp_create_nonce( self::ACTION_GET_SYNC_STATUS ),
+			'sync_in_progress'                => Sync::is_sync_in_progress(),
 			'excluded_category_ids'           => facebook_for_woocommerce()->get_integration()->get_excluded_product_category_ids(),
 			'excluded_tag_ids'                => facebook_for_woocommerce()->get_integration()->get_excluded_product_tag_ids(),
 			'i18n'                            => [
 				/* translators: Placeholders %s - html code for a spinner icon */
 				'confirm_resync'                => esc_html__( 'Your products will now be resynced to Facebook, this may take some time.', 'facebook-for-woocommerce' ),
 				'confirm_sync'                  => esc_html__( "Facebook for WooCommerce automatically syncs your products on create/update. Are you sure you want to force product resync?\n\nThis will query all published products and may take some time. You only need to do this if your products are out of sync or some of your products did not sync.", 'facebook-for-woocommerce' ),
-				'sync_in_progress'              => sprintf( esc_html__( 'Syncing... Keep this browser open until sync is complete. %s', 'facebook-for-woocommerce' ), '<span class="spinner is-active"></span>' ),
+				'sync_in_progress'              => sprintf( esc_html__( 'Syncing... %s', 'facebook-for-woocommerce' ), '<span class="spinner is-active"></span>' ),
```

Comment:
Maybe we could add a tooltip or something so that the users know they can close the page now. Just a thought.

Reply:
Yeah that may be a nice enhancement - opened a story for the future https://app.clubhouse.io/skyverge/story/55573/let-folks-know-they-can-leave-the-sync-page-now